README.md

```diff
@@ -57,6 +57,17 @@ $ yarn start
 
 The final `yarn start` command should open a local instance of Backstage in your browser, otherwise open one of the URLs printed in the terminal.
 
+### (Optional)Try on Docker
+
+Run the following commands if you have Docker environment
+
+```bash
+$ docker build . -t spotify/backstage
+$ docker run --rm -it -p 80:80 backstage
```

Comment:
Should image be `spotify/backstage` on this line as well?

Reply:
Just adding my 2 cents, but yup. Definitely should be `spotify/basckstage` as it currently won't find the image